tests/test_tasks/test_task_functions.py

```diff
@@ -42,6 +42,12 @@ def test__get_estimation_procedure_list(self):
         self.assertIsInstance(estimation_procedures[0], dict)
         self.assertEqual(estimation_procedures[0]['task_type_id'], 1)
 
+
+    def test_list_clustering_task(self):
+        # as shown by #383, clustering tasks can give problems to server
+        openml.config.server = self.production_server
+        openml.tasks.list_tasks(task_type_id=5, size=10)
```

Comment:
Hm, what's the expected outcome here? That is passes at all?

Reply:
Added